src/concepts/config-intro.md

```diff
@@ -139,7 +139,7 @@ bindings:
   tls_server:
     type: tls
     kind: server
-    vault: your_servers
+    vault: my_servers
```

Comment:
Looks good. 👍 

Btw, several other config files need to also change from `your_servers` to `my_servers` for consistency.

Reply:
right, thank you!